### PR TITLE
auto_init: fixup for #5185

### PR DESCRIPTION
**Pull request by OlegHahm**

### CODE REVIEW

File: sys/auto_init/netif/auto_init_encx24j600.c [hunk 1]
```diff
@@ -44,7 +44,11 @@ void auto_init_encx24j600(void)
 {
     DEBUG("auto_init_encx24j600(): initializing device...\n");
     /* setup netdev2 device */
-    encx24j600_setup(&encx24j600, ENCX24J600_SPI, ENCX24J600_CS, ENCX24J600_INT);
+    encx24j600_params_t p;
+    p.spi       = ENCX24J600_SPI;
+    p.cs_pin    = ENCX24J600_CS;
+    p.int_pin   = ENCX24J600_INT;
+    encx24j600_setup(&encx24j600, &p);
```
Comment (authmillenon) [inline sys/auto_init/netif/auto_init_encx24j600.c]: Unrelated, but the param struct should be board-defined here too, IMHO (unless I'm overlooking something).

**Comment by authmillenon** [comment]
ACK and go when Murdock is happy.

File: sys/auto_init/netif/auto_init_encx24j600.c (hunk 1 above)
Comment (OlegHahm) [inline sys/auto_init/netif/auto_init_encx24j600.c]: Do we have a board with this transceiver?

Comment (authmillenon) [inline sys/auto_init/netif/auto_init_encx24j600.c]: If not, how do we have a driver for it. If it is something pluggable, the `x_params.h` should reside in the application's directory.

Comment (kaspar030) [inline sys/auto_init/netif/auto_init_encx24j600.c]: There's no boatd. this chip is used on PoElli shields.

Comment (authmillenon) [inline sys/auto_init/netif/auto_init_encx24j600.c]: Then there should be some xbee-like configuration, or we find something better. The current way makes usage of multiple devices of that type impossible.

Comment (OlegHahm) [inline sys/auto_init/netif/auto_init_encx24j600.c]: This should be discussed in a separate issue.

Comment (authmillenon) [inline sys/auto_init/netif/auto_init_encx24j600.c]: Let me refer you to https://github.com/RIOT-OS/RIOT/issues/4672 ;-)

Comment (OlegHahm) [inline sys/auto_init/netif/auto_init_encx24j600.c]: Thanks.

**Pull request merged**